### PR TITLE
SSL Support

### PR DESCRIPTION
**Pull request by jtopjian**
This commit adds options to help with SSL connectivity:

* insecure: ignore SSL warnings and errors. useful for self-signed certs.
* cacert_file: specify the path or contents of a ca certificate.

### CODE REVIEW

File: cobbler/provider.go [hunk 1]
```diff
@@ -30,6 +30,20 @@ func Provider() terraform.ResourceProvider {
...
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: envDefaultFunc("COBBLER_INSECURE"),
+				Description: "Ignore SSL certificate warnings and errors.",
+			},
+
+			"cacert_file": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: envDefaultFunc("COBBLER_CACERT"),
```
Comment (tombuildsstuff) [inline cobbler/provider.go]: should this be `COBBLER_CACERT_FILE`?

**Review by tombuildsstuff — APPROVED** [review]
one minor 🤔  - otherwise LGTM 👍

File: cobbler/provider.go (hunk 1 above)
Comment (jtopjian) [reply cobbler/provider.go]: Yep, I'll get this fixed right now.

**Comment by jtopjian** [comment]
@tombuildsstuff Updated + added docs for the new settings.

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍

**Pull request merged**